src/backend/crossterm.rs

```diff
@@ -247,7 +482,7 @@ where
     }
 
     fn size(&self) -> io::Result<Rect> {
```

Comment:
breaking change independent of this PR: use Size rather than Rect?

Reply:
noted